cmd/juju/model/defaultscommand.go

```diff
@@ -2,18 +2,31 @@
 // Licensed under the AGPLv3, see LICENCE file for details.
 package model
 
+// tests to add
+// Test for no arg
+// Test for region only
+// Test for cloud only - fail because unknown key -- just cloud is invalid
+// Test for cloud/region - valid
+// TEst for cloud/region - invalid
+
 import (
 	"bytes"
 	"io"
 	"sort"
 	"strings"
 
+	names "gopkg.in/juju/names.v2"
```

Comment:
Not sure how that happened. I usually rely on goimports or paste from elsewehere. Removed